modules/ocf/files/shell/config.fish

```diff
@@ -0,0 +1,38 @@
+# Quit if not interactive
+if not status --is-interactive
+    exit
+end
```

Comment:
i don't know anything about fish, but if exit means the same thing as it does in sh, i wonder if this causes the entire shell to exit? lol

Reply:
oh shit you're right lmao

Reply:
oh wait https://fishshell.com/docs/current/commands.html#exit

`If exit is called while sourcing a file (using the source builtin) the rest of the file will be skipped, but the shell itself will not exit.`

Does that seem to be intended behavior?

Reply:
yeah, that sounds right (different than the sh behavior, but that's ok)